_posts/2024-02-06-jahresrückblick.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title:  "Jahresrückblick 2023 und Ausblick für 2024"
+categories: verein
+---
+Eigentlich hatten wir geplant, den Jahresrückblick zu verfassen, sobald alle derzeit noch laufenden Projekte abgeschlossen wären. Nun ist bereits der Februar angebrochen, und es herrschte eine zu lange Stille.
+
+## Finanzen
+Kurz und bündig: Wir haben das Jahr mit einem Gewinn von XXX€ abgeschlossen und stehen finanziell weiterhin sehr solide da. Es war uns sogar möglich, zusätzliche Investitionen zu tätigen - nähere Details dazu folgen in Kürze.
+
+## Die Instanz
+Das Jahr begann mit einem Aufschwung, der durch Twitter/X ermöglicht wurde. Daher ist es nicht verwunderlich, dass wir in den Statistiken bezüglich der aktiven Accounts einen allmählichen Rückgang verzeichnen. Dennoch nutzen wöchentlich mehr als 3000 Personen unsere Instanz. Angesichts der Tatsache, dass wir keine spezifischen Marketingaktivitäten betreiben, sind diese Ergebnisse im sechsten Jahr unseres Bestehens äußerst positiv.
+
+Für Aufsehen sorgte die Nachricht, dass Threads eine ActivityPub-Schnittstelle erhalten und dadurch Teil des Fediverse werden könnte - vorausgesetzt, die Nutzer*innen von Threads geben ihre Zustimmung. Wir haben als Instanz beschlossen, keine vorschnellen Entscheidungen zu treffen und werden Threads wie jede andere Instanz begegnen. Sollte es Ereignisse geben, die mit unseren Richtlinien unvereinbar sind, werden wir entsprechend reagieren.
+Ein weiteres wichtiges Thema ist die Entwicklung rund um AI/ChatGPT. Kurz gefasst: Wir haben unsere Instanz für diese Technologien gesperrt. ChatGPT kann beispielsweise keinen Zugriff auf unsere Profile nehmen.
```

Comment:
Hmm, in unserer robots.txt steht der Default drin und das ist "nur" GPTBot (https://norden.social/robots.txt). Machen wir sonst noch was, das ich nicht auf dem Schirm hab?

Reply:
GPTBot ist aber _der_ ChatGPT-Bot.

In diesem Doc gibt es noch Google-Extended für/gegen Bard:
https://www.eff.org/deeplinks/2023/12/no-robotstxt-how-ask-chatgpt-and-google-bard-not-use-your-website-training